pwnlib/asm.py

```diff
@@ -199,6 +199,7 @@ def which_binutils(util):
             else:
                 patterns = ['%s*linux*-%s' % (arch, gutil),
                             '%s*-elf-%s' % (arch, gutil),
+                            '%s-none*-%s' % (arch, gutil),
```

Comment:
What if you have both packages installed? Does this break anything or it passes the tests? Also, IIRC bare-metal ARM can be compiled with any `*arm*-gcc` with correct commandline flags, not only by `arm-none-eabi-gcc`

Reply:
Based on what I have seen from my own testing (and manually tested), `which_binutils()` returns the first binutils that it recognizes. It will first find `linux-gnueabi` then the `linux-gnueabihf`. If those are not there then it tries to find `none-eabi`. There seems to be no conflict otherwise.

Reply:
Based on what I have seen from my own testing (and manually tested), which_binutils() returns the first binutils that it recognizes. It will first find `linux-gnueabi` then the `linux-gnueabihf`. If those are not there then it tries to find `none-eabi`. There seems to be no conflict.